docs/yaml/spec/v2.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Read the Docs configuration file",
+  "type": "object",
+  "required": [],
+  "properties": {
+
+    "version": {
+      "description": "The version of the spec to use",
+      "type": "string",
+      "enum": ["1", "2"],
+      "default": "1"
+    },
+
+    "formats": {
+      "description": "Formats of the documentation to be built",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": ["htmlzip", "pdf", "epub", "singlehtmllocalmedia"]
```

Comment:
I added the `singlehtmllocalmedia` option regarding to #3220 (a little long by the way)